### PR TITLE
Remove Latest Photos module from RightRail

### PR DESCRIPTION
**Pull request by bognix**

### CODE REVIEW

File: extensions/wikia/LatestPhotos/LatestPhotosController.class.php [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+	 *
+	 * @author Andrzej 'nAndy' Łukaszewski
+	 */
+	const THUMB_SIZE = 82;
+
+	/**
+	 * Just a string concatenated with other in creation of Memc Key
+	 * @var String
+	 */
+	const MEMC_KEY_VER = '1.1';
```
Comment (vforge) [inline extensions/wikia/LatestPhotos/LatestPhotosController.class.php]: Please bump

**Comment by vforge** [comment]
lgtm

File: extensions/wikia/LatestPhotos/LatestPhotos.setup.php [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * LatestPhotos Extension created based on Oasis module
```
Comment (hakubo) [inline extensions/wikia/LatestPhotos/LatestPhotos.setup.php]: Where is it being used now?

Comment (bognix) [inline extensions/wikia/LatestPhotos/LatestPhotos.setup.php]: LightboxHelper.class.php

**Pull request merged**